src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json

```diff
@@ -0,0 +1,245 @@
+{
+    "value": [
+        {
+            "id": null,
+            "name": "dotnet",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=LinuxFunctions",
+            "properties": {
+                "name": "dotnet",
+                "display": ".NET Core",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "3.1",
+                        "runtimeVersion": "dotnet|3.1",
+                        "supportedFunctionsExtensionVersions": [
+                            "~3"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {
+                            "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+                        },
+                        "siteConfigPropertiesDictionary": {
+                            "Use32BitWorkerProcess": false,
+                            "linuxFxVersion": "dotnet|3.1"
+                        },
+                        "isPreview": false,
+                        "isDeprecated": false,
+                        "isHidden": false
+                    },
+                    {
+                        "displayVersion": "2.2",
```

Comment:
what is the default extension version if --default-Version is not specified?

Reply:
I might be misunderstanding what you're saying, but right now the default extension version in the CLI is 2 if they don't specify the --functions-version argument (see [this](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L2573-L2576)). This is something I've been meaning to bring up to see if that needs to be changed. I believe we did that when v3 was very new.